spec/service_models/miq_ae_service_service_spec.rb

```diff
@@ -81,6 +83,7 @@ def invoke_ae
 
         @parent.reload
         expect(@parent.direct_service_children).to eq([])
+        expect(@parent.service_resources).to eq([])
       end
 
       it "validates the parent service" do
```

Comment:
@d-m-u The changes look good. Could we add a test for when the child Service being added to the Service already has a parent Service? We would expect it to raise an error.